src/test/scala/cromwell/WdlThreeStepFixture.scala

```diff
@@ -0,0 +1,49 @@
+package cromwell
+
+import cromwell.binding.WdlBinding
+
+trait WdlThreeStepFixture {
```

Comment:
Is there an advantage to using this structure rather than simply declaring the binding as a val on the spec class?